test/mlir/test/gc/Transforms/GPU/lit.local.cfg

```diff
@@ -1,2 +1,2 @@
-if not config.gc_use_gpu:
+if not config.gc_use_imex:
```

Comment:
This subdirectory was originally designed for general GPU tests. Shall we rename this directory to IMEX? Or maybe we can add a new subdirectory in test/gc/transforms/GPU/IMEX ?

Reply:
Right, so it should either be less strict and allow some of the tests to run. Another option is to split the tests into different folders. Let's decide once we have them.